test/browser_tests/cucumber/step_definitions/feedback-form.js

```diff
@@ -0,0 +1,90 @@
+const { Then, When, Before } = require( 'cucumber' );
+const chai = require( 'chai' );
+const expect = chai.expect;
+const chaiAsPromised = require( 'chai-as-promised' );
+
+const BASE_SEL = '.o-feedback';
+const FORM = BASE_SEL + ' .o-form';
+const COMMENT_FIELD = BASE_SEL + ' #comment';
+const RADIO_BUTTON_1 = BASE_SEL + ' #is_helpful_1';
+const SUBMIT_BUTTON = BASE_SEL + ' [type="submit"]';
+const NOTIFICATION = BASE_SEL + ' .m-notification';
+const SUCCESS_CLASS = 'm-notification__success';
+const ERROR_CLASS = 'm-notification__error';
+
+chai.use( chaiAsPromised );
+
+let _dom;
+
+Before( function() {
```

Comment:
TRIVIAL: You could use arrow functions here and throughout to make the tests more compact `() => { …`